src/components/Card/styles.module.css

```diff
@@ -1,58 +1,63 @@
 .root {
-  width: 350px;
-  height: 500px;
-  margin: 10px;
+    width: 350px;
+    height: 500px;
+    margin: 10px;
```

Comment:
Remove changes not required and adhere to the proper indentation according to rest of the project